Subject: Mention that the apple filetype/creator is only available for entries that (...)
Origin: FILE5_35-55-g642f269e <https://github.com/file/file/commit/FILE5_35-55-g642f269e>
Upstream-Author: Christos Zoulas <christos@zoulas.com>
Date: Mon Feb 18 18:59:25 2019 +0000

    Mention that the apple filetype/creator is only available for entries that
    have it (Kamil Dudka)

--- a/doc/file.man
+++ b/doc/file.man
@@ -171,6 +171,9 @@
 used by older MacOS versions.
 The code consists of eight letters,
 the first describing the file type, the latter the creator.
+the first describing the file type, the latter the creator.
+This option works properly only for file formats that have the
+apple-style output defined.
 .It Fl b , Fl Fl brief
 Do not prepend filenames to output lines (brief mode).
 .It Fl C , Fl Fl compile
